src/lib/components/grid/Grid.tsx

```diff
@@ -1,24 +1,171 @@
 import React from "react";
 import classNames from "classnames";
 import { FlexSpacing } from "../flex/types";
+import { ResponsiveGridValue } from "./types";
 
 export const COLUMNS = [1, 2, 3, 4] as const;
 export type Columns = (typeof COLUMNS)[number];
 
+const GRID_ALIGN_ITEMS = ["start", "end", "center", "stretch", "baseline"] as const;
+type GridAlignItems = (typeof GRID_ALIGN_ITEMS)[number];
+
+const GRID_JUSTIFY_ITEMS = ["start", "end", "center", "stretch"] as const;
+type GridJustifyItems = (typeof GRID_JUSTIFY_ITEMS)[number];
+
+const GRID_ALIGN_CONTENT = [
+  "start",
+  "end",
+  "center",
+  "stretch",
+  "space-around",
```

Comment:
I just realize we camel-case these kinds of values in the `Flex` components, so let's do the same here, e.g. "spaceAround".